manifests/gcp_marketplace/guide.md

```diff
@@ -0,0 +1,65 @@
+# Deploy Kubeflow Pipelines from Google Cloud Marketplace
+
+You can go to [Google Cloud Marketplace](https://console.cloud.google.com/marketplace) to deploy Kubeflow Pipelines via UI.
+You can find the instances in [Pipelines Console](http://pantheon.corp.google.com/ai-platform/pipelines).
+
+## Cluster
+You can choose an existing cluster or create a new cluster. Creating a new cluster from Marketplace config page doesn't allow
```

Comment:
Suggest: You can choose an existing cluster for Kubeflow Pipelines deployment, or create a new cluster.